### ISSUE TITLE
[iOS] Automatically set entitlements

### ISSUE THREAD

**Issue by chancezeus** [OneSignal/OneSignal-Cordova-SDK#115]
I really love your platform and like your plugin, it is easy to use and works properly. There is just one "problem". If you build your project from the command line the resulting ipa is missing the necessary entitlements. Looking at another cordova push plugin they have the following in their plugin.xml:
```
    <config-file target="*-Debug.plist" parent="aps-environment">
      <string>development</string>
    </config-file>
    <config-file target="*-Release.plist" parent="aps-environment">
      <string>production</string>
    </config-file>
```
This piece of code automatically inserts the correct aps-environment into the entitlement files for both debug and release. I'm wondering if this plugin could do the same.

**Comment by jkasten2** [comment]
@chancezeus Thanks for the suggestion! I'll add it in our next release.

**Comment by jkasten2** [comment]
@chancezeus Just added this in commit a76b3721382bd23c1ddde98b2407bcd87a0de512 and it is now released in our 2.0.8 version.
